src/components/Dashboard/Content/index.tsx

```diff
@@ -47,22 +49,26 @@ const DashboardContent = ({
       </div>
     </div>
   ) : (
-    <div className="flex flex-col h-full" data-test-id={dashboardDataTestIds.content}>
+    <div className="flex flex-col h-full" data-test-id={dashboardContentDataTestIds.base}>
       <div
-        style={{ backgroundImage: `url(${getHighResolutionImage(surveys[currentPosition].coverImageUrl)})` }}
+        style={{
+          backgroundImage: `url(${surveys.length ? getHighResolutionImage(surveys[currentPosition].coverImageUrl) : ''})`,
+        }}
         className="w-full h-[302px] rounded-[12px] bg-cover duration-500 ease-in-out"
       ></div>
       <div className="flex flex-row justify-between mt-[38px]">
         <div className="flex flex-col justify-between">
           <p className="text-white text-x-regular font-extrabold">{surveys[currentPosition].title}</p>
           <p className="text-white text-regular tracking-survey-tight opacity-60 mt-2">{surveys[currentPosition].description}</p>
         </div>
-        <button
-          type="button"
-          className="w-[56px] h-[56px] bg-white rounded-full inline-flex items-center justify-center text-black-chinese"
-        >
-          <ArrowRight />
-        </button>
+        <Link to={`surveys/${_get(surveys, `${currentPosition}.id`, '')}`}>
```

Comment:
Should we create a new function to get the survey by the `currentPosition` and another function to get the path of a survey?
This can extract the logic, and later you can change the path in one place instead of finding these `surveys/${_get...` texts

Reply:
@hoangmirs updated in [8bf88ce](https://github.com/manh-t/react-survey/pull/61/commits/8bf88ced41b41cbe08f1459924d5efd174dbdbb7)